constants.yaml

```diff
@@ -0,0 +1,17 @@
+# Build Directories
```

Comment:
As its currently set up, correct. This is easy to change for any given project though.